app/models/auth_token.rb

```diff
@@ -2,32 +2,33 @@ class AuthToken < ActiveRecord::Base
 
   belongs_to :user
 
-  validates :authentication_token, presence: true, uniqueness: true
+  validates :encrypted_authentication_token, presence: true, uniqueness: true
```

Comment:
Remove uniqueness constraint - not needed... plus the encrypted value being unique does not mean the tokens are different